assets/section-main-product.css

```diff
@@ -595,15 +604,19 @@ a.product__text {
   .grid__item.product__media-item--full {
     width: 100%;
   }
+
+  .product--columns .product__media-item:not(.product__media-item--variant) {
+    max-width: calc(50% - var(--grid-desktop-horizontal-spacing) / 2);
+  }
```

Comment:
Yeah I was thinking about it a bit more and I think because we do offer the option to choose how big the media column is, I think we're providing all the options for the merchant to make the best decision for their type of products and medias. 
So I think I'm ok with it like this because there are ways around it that are straight forward.